### ISSUE TITLE
App Store asks: Does your app use encryption?

### ISSUE THREAD

**Issue by Piiit** [noi-techpark/it.bz.noi.community.ios#20]
I think we can safely say NO here, right?

![image](https://user-images.githubusercontent.com/6096063/133983877-a87ee251-7a7f-48cb-b9f5-44bb7c71f698.png)

**Comment by matax87** [comment]
@Piiit yes, I can say NO. Moreover, I'm going to provide a PR to fix this for future uploads.
